modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/repository_entry_browser/js/ItemSelectorRepositoryEntryBrowser.es.js

```diff
@@ -87,7 +86,7 @@ class ItemSelectorRepositoryEntryBrowser extends PortletBase {
 
 		const items = itemsNodes.map((node) => node.dataset);
 
-		const clickableItems = Array.from(this.all('.icon-view'));
+		const clickableItems = Array.from(this.all('.icon'));
```

Comment:
I don't think we should change this selector. It must be `'.icon-view'` because that is the class for the view icon, and this should be the reason why tests are failing like @Hanlf mentioned.